README.md

```diff
@@ -118,3 +119,19 @@ scraper.scrape({
   maxDepth: 1
 }).then(console.log).catch(console.log);
 ```
+
+#### Example 3. Recursive downloading of entire website
```

Comment:
It is hard to see difference between Example 3 and Example 2

I suggest to make this example only for `urlFilter` (without `recursive` and `maxDepth`) to highlight this feature 
For example, title may be like **Filtering out external resources**